lib/grell/rawpage.rb

```diff
@@ -36,6 +35,17 @@ def has_selector?(selector)
       page.has_selector?(selector)
     end
 
+    def wait_for_all_ajax_requests(timeout, interval)
+      Timeout::timeout(timeout) do
+        (timeout / interval).ceil.times do
+          jquery_active = page.evaluate_script('(typeof(jQuery)!="undefined") && jQuery.active;')
```

Comment:
this should be `!==` and btw typeof doesn't need `()` so you could write
```
typeof jQuery !== "undefined" && jQuery.active;
```

(untested in this context but it's legit JS syntax)